x-pack/legacy/plugins/apm/public/components/shared/Summary/ResultSummaryItem.tsx

```diff
@@ -12,7 +12,7 @@ interface ResultProps {
   result: string;
 }
 
-export function Result({ result }: ResultProps) {
+export function ResultSummaryItem({ result }: ResultProps) {
```

Comment:
`result` is a bit generic. What do you think about `transactionResult` to be more clear?

Reply:
Fixed. Also renamed the component to `TransactionResultSummaryItem`.